src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -3,7 +3,7 @@
 /* eslint-disable import/no-unresolved */
 
 import * as React from 'react';
-import {FlatList, KeyboardAvoidingView, Platform, View} from 'react-native';
+import {FlatList, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `KeyboardAvoidingView` has been removed, which is consistent with the changes made in the component's usage. This change seems to be part of a larger refactor where keyboard behavior handling is being moved to a separate component (`KeyboardWrapper`). 
```diff
- import {FlatList, KeyboardAvoidingView, Platform, View} from 'react-native';
+ import {FlatList, View} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->